tests/indent/zig_spec.lua

```diff
@@ -18,5 +18,18 @@ describe("indent Zig:", function()
   describe("new lines:", function()
     run:new_line("pr-3269.zig", { on_line = 5, text = "return;", indent = 4 })
     run:new_line("pr-3269.zig", { on_line = 6, text = "", indent = 0 })
+    run:new_line("types.zig", { on_line = 4, text = "z: f32,", indent = 4 })
+    run:new_line("types.zig", { on_line = 7, text = "", indent = 0 })
+    run:new_line("types.zig", { on_line = 9, text = "spherical,", indent = 4 })
+    run:new_line("types.zig", { on_line = 12, text = "", indent = 0 })
+    run:new_line("types.zig", { on_line = 14, text = "predicate: bool,", indent = 4 })
+    run:new_line("types.zig", { on_line = 17, text = "", indent = 0 })
+    run:new_line("types.zig", { on_line = 22, text = "pub fn test(ptr: *Opaque) {}", indent = 4 })
+    run:new_line("types.zig", { on_line = 23, text = "", indent = 0 })
+    run:new_line("types.zig", { on_line = 21, text = "const copy = ptr;", indent = 8 })
+    run:new_line("pr-7199.zig", { on_line = 4, text = "{", indent = 4 })
```

Comment:
This one is failing, possibly because the tree takes the ending `}` and set it to be a block, instead of considering it a singular error node inside the original block